_data/education.yml

```diff
@@ -27,11 +27,12 @@ websites:
       tfa: No
 
     - name: Babbel
-      url: https://www.babbel.com
+      url: https://uk.babbel.com/
```

Comment:
babbel.com redirects to www.babbel.com for me.

Reply:
The subdomain appears to resolve based on the browser's language preferences. I'll undo the `url` change.